.github/CODEOWNERS

```diff
@@ -1,16 +1,20 @@
 # Documentation/Infrastructure
-/docs                                       @JustinyAhin @felixarntz
 /admin                                      @JustinyAhin @felixarntz
 /tests/admin                                @JustinyAhin @felixarntz
 /bin                                        @JustinyAhin @felixarntz
 /.github                                    @JustinyAhin @felixarntz
 
+# Focus: Database
+/modules/database
+/tests/modules/database
+/tests/testdata/modules/database
+
 # Focus: Images
 /modules/images                             @adamsilverstein @getsource
 /tests/modules/images                       @adamsilverstein @getsource
 /tests/testdata/modules/images              @adamsilverstein @getsource
 
-# Focus: JavaScript
+# Focus: CSS & JS
```

Comment:
I think here is where I saw the "CSS & JS", but you mentioned it should be "JS & CSS" :)

Reply:
@felixarntz Fixed!